app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -62,7 +64,21 @@ export default class SingleAddon extends React.Component{
             {app.appDetails && app.appDetails.description}
           </h5></div>
           {updatesVersion ? <span className="update-notification">New version Available {updatesVersion.version} </span>: null}
-          {updatesVersion ? <span className="glyphicon glyphicon-download-alt download-update" onClick={(e) => this.getUpdate(e, updatesVersion.uid)} />: null}
+          {
+            updatesVersion ?
+              <span
+                className="glyphicon glyphicon-download-alt download-update"
+                onClick={(event) => this.getUpdate(event, updatesVersion.uid)} />
+              : null}
+          {
+            updatesVersion ?
+              <span
+                className="btn btn-secondary"
+                id="upgrade-btn"
+                onClick={(event) => handleUpgrade(app, updatesVersion.version, updatesVersion.uid)}>Upgrade
+              </span>
+              : null
+          }
         </td>
```

Comment:
Looks like two different conditions braches checking the same thing `updatesVersion`. You could just combine them into one